src/Uno.UI/UI/Xaml/Controls/Control/Control.cs

```diff
@@ -164,12 +165,38 @@ internal View TemplatedRoot
 					{
 						RegisterContentTemplateRoot();
 
-						OnApplyTemplate();
+						if (FeatureConfiguration.Control.UseDeferredOnApplyTemplate)
+						{
+							// It's too soon the call the .ApplyTemplate method: it should be invoked right before the Loading event.
```

Comment:
```suggestion
							// It's too soon to call the .ApplyTemplate method: it should be invoked after the Loading event.
```

right?

Reply:
Actually it's the call to ".OnApplyTemplate".  I'll fix it.  Nice catch!